mcobs_eu_demo.postman_collection.json

```diff
@@ -0,0 +1,1638 @@
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var mandateId = pm.response.json().mandateId;",
+									"pm.environment.set(\"mandateId\", mandateId);",
+									"",
+									"var paymentId = pm.response.json().paymentId;",
+									"pm.environment.set(\"paymentId\", paymentId);",
+									"",
+									"var redirectUrl = pm.response.json().flowUrl;",
+									"var flowId = redirectUrl.split(\"/\").last();",
+									"pm.environment.set(\"flowId\", flowId);"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "Accept",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"destinationId\": \"{{destinationId}}\",\n    \"paymentRail\": \"UkFasterPayments\",\n    \"currency\": \"GBP\",\n    \"redirectUrl\": \"{{redirectUrl}}\",\n    \"limits\": {\n        \"maxAmountPerPayment\": 100.0,\n        \"maxAmountPerMonth\": 1000.0\n    },\n    \"endUser\": {\n        \"id\": \"0001789937234\"\n    }\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrlPayments}}/mandates",
+							"host": [
+								"{{baseUrlPayments}}"
+							],
+							"path": [
+								"mandates"
+							]
+						}
+					},
+					"response": [
+						{
+							"name": "Success",
+							"originalRequest": {
+								"method": "POST",
+								"header": [
+									{
+										"description": "Added as a part of security scheme: bearer",
+										"key": "Authorization",
+										"value": "Bearer <token>"
+									},
+									{
+										"key": "Accept",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n  \"amount\": \"<double>\",\n  \"currency\": \"<string>\",\n  \"destinationId\": \"<uuid>\",\n  \"redirectUrl\": \"<string>\",\n  \"schemeId\": \"<string>\",\n  \"mandateId\": \"<uuid>\",\n  \"date\": \"<dateTime>\",\n  \"providerId\": \"<string>\"\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{baseUrl}}/v2/payments",
+									"host": [
+										"{{baseUrl}}"
+									],
+									"path": [
+										"v2",
+										"payments"
+									]
+								}
+							},
+							"status": "OK",
+							"code": 200,
+							"_postman_previewlanguage": "json",
+							"header": [
+								{
+									"key": "Content-Type",
+									"value": "application/json"
+								}
+							],
+							"cookie": [],
+							"body": "{\n    \"redirectUrl\": \"https://connectplatform.test.aiia.eu/start/storage-payments/cafc238f-2fba-4387-8a96-0a8f22c50478\",\n    \"mandateId\": \"662bea2c-a0af-42db-960f-3f7bb3cc755a\"\n}"
```

Comment:
We have the test environment for connect here in a bunch of places. Also do the other collections include sample responses?

Reply:
Looks like no - other collection does not have sample responses. I will remove them.